src/utils/globals.ts

```diff
@@ -26,9 +30,17 @@ export const tempDIR = "tempGitLocations" as string;
 export const path = "./github/workflows" as string;
 export const inform = Debug("ghas:inform") as Debug.Debugger;
 export const error = Debug("ghas:error") as Debug.Debugger;
-export const destDir = "Desktop" as string;
-export const windestDir = "Documents" as string;
-export const user = process.cwd().split("/")[2] as string;
-export const winUser = process.cwd().split("\\")[2] as string;
 export const reposFileLocation = "./bin/repos.json" as string;
 export const orgsFileLocation = "./bin/organizations.json" as string;
+export const platform = os.platform() as string;
+export const destDir =
+  platform === "win32" ? ("Documents" as string) : ("Desktop" as string);
```

Comment:
Does `Desktop` not exist on Windows? I always wanted to keep this the same across OS's. 

Is there a reason why for Windows we are using Documents 🤔

Reply:
It does, I was just keeping with what you already had. Didn't know if you have reason to use Documents to begin with. It can/should be consolidated... Desktop would work just fine imho.

Reply:
If you could move it to Desktop that would be amazing 🙇